controls/ubuntu-18.04-server-cis-4.1.1.1.rb

```diff
@@ -23,14 +23,16 @@
   "
   impact 0.7
   tag severity: "high"
-  tag gtitle: nil
-  tag gid: nil
-  tag rid: nil
-  tag stig_id: nil
-  tag fix_id: nil
-  tag cci: nil
-  tag nist: ["AU-12", "AU-3", "Rev_4"]
+  tag nist: ["AU-12", "AU-3"]
   tag cis_level: 2
-  tag cis_controls: ["6.2", "6.3", "Rev_7"]
+  tag cis_controls: ["6.2", "6.3"]
```

Comment:
can we keep the cis CDC revision as a tag perhaps?

` tag cis_cdc_version: '7'`

That way when we display the results in Heimdall we can have it auto-generate a link to the control description on the cis website.